app/services/volunteer_op_marker_builder.rb

```diff
@@ -0,0 +1,40 @@
+class VolunteerOpMarkerBuilder
+  def initialize(organisations, listener, marker_builder, url_helper)
+    @organisations = organisations
+    @marker_builder = marker_builder
+    @listener = listener
+    @url_helper = url_helper
+  end
+
+  def perform
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for perform is too high. [17/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [22/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)